### PR TITLE
Add paperless

### PR DESCRIPTION
**Pull request by highghlow**
App request: #1035 

# App Submission

### App name
Paperless

### 256x256 SVG icon
_(Submit an icon with no rounded corners as it will be dynamically rounded with CSS. GitHub doesn't allow uploading SVGs directly, so please upload your icon to an alternate service, like https://svgur.com, and paste the link below.)_
_We will help finalize this icon before the app goes live in the Umbrel App Store._

Again, that will be a pain to make into an svg

paperless

### Gallery images
_(Upload 3 to 5 high-quality gallery images (1440x900px) of your app in PNG format, or just upload 3 to 5 screenshots of your app and we'll help you design the gallery images.)_
_We will help finalize these images before the app goes live in the Umbrel App Store._

proxy-image3
proxy-image2
proxy-image

### I have tested my app on:
- [ ] umbrelOS on a Raspberry Pi
- [ ] umbrelOS on an Umbrel Home
- [x] umbrelOS on Linux VM

### CODE REVIEW

**Review by pavlojs — CHANGES_REQUESTED** [review]
I think we need to stick to specified version with sha256 provided

`image: ghcr.io/paperless-ngx/paperless-ngx:2.7.2@sha256:703c990a790dfd4d25fb56df3afec27b13cb0926a3818bf265edac9c71311647`

File: paperless/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      APP_PORT: 8000
+
+  broker:
+    image: docker.io/library/redis:7
+    restart: unless-stopped
+    volumes:
+      - ${APP_DATA_DIR}/redisdata:/data
+
+  webserver:
+    image: ghcr.io/paperless-ngx/paperless-ngx:latest
```
Comment (pavlojs) [inline paperless/docker-compose.yml:16]: `image: ghcr.io/paperless-ngx/paperless-ngx:2.7.2@sha256:703c990a790dfd4d25fb56df3afec27b13cb0926a3818bf265edac9c71311647`

Comment (highghlow) [reply paperless/docker-compose.yml]: Yeah, sorry, I forgot to pin the version, thanks for pointing that out. Fixed it!

**Comment by nmfretz** [comment]
Thanks for another submission of a requested app @highghlow! 

I've gone through and made the following changes to get it working. Can you please take a look through and give it a quick test? Everything works for me:

- removed some accidentally committed files from the paperless data dir
- committed all the data subdirectories to ensure proper ownership
- added postgres as the database (recommended by Paperless)
- added an exports.sh to ensure that the `consume` and `export` directories in the main Umbrel data directory are created with the correct ownership
- Added environment variables to create a super user and included default credentials in the app manifest. Without a superuser, you can't log in to Paperless after install unless you run a super user creation command inside the container, which we don't want users to have to do.
- Fixed the Tika digest. It was pointing to a non-existent manifest so was resulting in a failed install.

We'll get gallery assets made and then send Paperless-ngx to the app store!

**Comment by highghlow** [comment]
> * removed some accidentally committed files from the paperless data dir

I think I added them to resolve some username+password issue...

> * Added environment variables to create a super user and included default credentials in the app manifest. Without a superuser, you can't log in to Paperless after install unless you run a super user creation command inside the container, which we don't want users to have to do.

Yes, that's was the issue. Thanks for fixing it in a non-terrible way!

(how do you add emojis, anyway, hooray_emoji)

**Comment by nmfretz** [comment]
Thanks again for submitting Paperless-ngx @highghlow! Gallery assets are here: https://github.com/getumbrel/umbrel-apps-gallery/tree/master/paperless

I have made a few commits to finalize the submission. Going live to the app store now.

**Pull request merged**